build.common.gradle

```diff
@@ -92,8 +92,8 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_7
-        targetCompatibility JavaVersion.VERSION_1_7
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
```

Comment:
Java 8?

Reply:
i just changed that to java 8, since it speeds up i/o and stuff i think, and also stuff like lambda expressions